src/FunctionLoader.cs

```diff
@@ -18,6 +18,7 @@ namespace Microsoft.Azure.Functions.PowerShellWorker
     internal class FunctionLoader
     {
         private readonly Dictionary<string, AzFunctionInfo> _loadedFunctions = new Dictionary<string, AzFunctionInfo>();
+        private const string _latestAzModulePath = @"D:\Program Files (x86)\ManagedDependencies\PowerShell\AzPSModules\1.0.0";
```

Comment:
In fact, const members all have the first letter upper case in this code base. You can search `private const` to see some examples. So here we should make it `LatestAzModulePath`.